ovos_phal_plugin_ipgeo/__init__.py

```diff
@@ -2,13 +2,15 @@
 from ovos_plugin_manager.phal import PHALPlugin
 from ovos_utils.configuration import get_webcache_location, LocalConf
 from ovos_utils.messagebus import Message
+from ovos_utils.log import LOG
+from urllib3.exceptions import NewConnectionError
 
 
 class IPGeoPlugin(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus, "ovos-phal-plugin-ipgeo", config)
-        self.location = {}
```

Comment:
this was meant to save repeat http calls, any reason to not store the value?

Reply:
I suppose not; any time the plugin is reloaded we'll get a fresh location, I think the only edge case would be updating on network change but that should be handled explicitly anyways